### PR TITLE
check and format db-checkbook and db-template files

### PR DESCRIPTION
**Pull request by damonmcc**
for products that are being checked, [here's](https://github.com/NYCPlanning/data-engineering/actions/runs/5764031074/job/15627122402) what it looks like when python code fails the format check on a PR

fyi [documentation](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html) on `black`

### CODE REVIEW

**Comment by alexrichey** [comment]
I suppose we could implement this at the product level. However, I think it might be more useful to pipe in the list of changed files, regardless of which project they belong to. (ie make this a repo-level check)

File: products/db-checkbook/build_scripts/build.py [hunk 1]
```diff
@@ -6,31 +6,36 @@
 
 from . import LIB_DIR, OUTPUT_DIR, SQL_QUERY_DIR
 
-DB_URL = 'sqlite:///checkbook.db'
+DB_URL = "sqlite:///checkbook.db"
```
Comment (athursland) [inline products/db-checkbook/build_scripts/build.py:9]: this is purely a comment  but reading [the current Black style documentation](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) is so thorough it's wild - never really thought about the aesthetic merits of favoring double quotes over single quotes for strings lol

Comment (fvankrieken) [reply products/db-checkbook/build_scripts/build.py:9]: If you want a rollercoaster of a discussion on it

https://github.com/psf/black/issues/118

Comment (damonmcc) [reply products/db-checkbook/build_scripts/build.py:9]: definitely thorough, imagine trying to make so many devs happy with a common style and (pretty much) succeeding!

**Pull request closed without merging**